### PR TITLE
Node/update

### PR DESCRIPTION
**Pull request by mjadach-iv**

### CODE REVIEW

File: src/components/Modal/node/SendMessageModal.tsx [hunk 1]
```diff
@@ -39,16 +39,17 @@ export const SendMessageModal = ({ peerId }: SendMessageModalProps) => {
...
+  const [directMessage, set_directMessage] = useState<boolean>(true);
   const [message, set_message] = useState<string>('');
   const [receiver, set_receiver] = useState<string>(peerId ? peerId : '');
   const [openModal, set_openModal] = useState<boolean>(false);
 
   const maxLength = 500;
   const remainingChars = maxLength - message.length;
 
-  const nonAutomaticPathTooltip = 'Disable `automatic path` to enable `Number of hops`';
+  const nonAutomaticPathTooltip = 'Disable `direct message` and `automatic path` to enable this field';
```
Comment (diegoalzate) [inline src/components/Modal/node/SendMessageModal.tsx:52]: when direct message is enabled, this tooltip does not show up

File: src/future-hopr-lib-components/Modal/styled.tsx [hunk 2]
```diff
@@ -1,11 +1,19 @@
 import styled from '@emotion/styled';
 import { Dialog, DialogContent, IconButton } from '@mui/material';
 
-export const SDialog = styled(Dialog)`
+export const SDialog = styled(({
+  width,
+  ...rest
+}: any) => <Dialog {...rest} />)`
```
Comment (diegoalzate) [inline src/future-hopr-lib-components/Modal/styled.tsx:7]: We should change this any in either this pr or another, the code does work but we should try to minimize the use of anys

**Review by diegoalzate — APPROVED** [review]

File: src/components/Modal/node/SendMessageModal.tsx (hunk 1 above)
Comment (mjadach-iv) [reply src/components/Modal/node/SendMessageModal.tsx:52]: Thx, updated

File: src/future-hopr-lib-components/Modal/styled.tsx (hunk 2 above)
Comment (mjadach-iv) [reply src/future-hopr-lib-components/Modal/styled.tsx:7]: updated

**Pull request merged**